Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:alpine AS build
+RUN apk add --no-cache git make
+WORKDIR /build
+COPY . .
+ENV CGO_ENABLED=0
+RUN make build
+
+FROM alpine:latest
+COPY --from=build /build/dist/yamlfmt /bin/yamlfmt
```

Comment:
Does this need `--chmod=755` along with the copy?

Reply:
Should this have `ENTRYPOINT ["/bin/yamlfmt"]` to make it so `docker run <container>` command can just take the yamlfmt args after? I don't think anything else but the yamlfmt binary will ever be run in this container.

Reply:
I do not think that the chmod flag is necessary due to the binary has the correct permissions after the build is finished.
I think you are right in specifying the yamlfmt binary as ENTRYPOINT for the image. I will adjust that in the next minutes.